x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -455,8 +456,16 @@ export class TaskRunner<
       }
     });
 
-    const { alertsToReturn, recoveredAlertsToReturn } =
-      this.legacyAlertsClient.getAlertsToSerialize();
+    let alertsToReturn: Record<string, RawAlertInstance> = {};
```

Comment:
Can we add a unit test for the state when `autoRecoverAlerts` is false?

Reply:
Resolved in this commit 9d7056ee03ffaa203f6891689e5aefeb8ffe3ab2